dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -205,6 +205,14 @@ jenkins:
               - envVar:
                   key: "JENKINS_JAVA_BIN"
                   value: "<%= agent['agentJavaBin'] ? agent['agentJavaBin'] : @jcasc['agents_setup']['ubuntu']['agentJavaBin'] %>"
+            <%- if agent['javahome'] -%>
```

Comment:
shouldn't it be javaHome ????

Reply:
You're right 😓 
Creating a PR to fix it